src/language/mongoDBService.ts

```diff
@@ -96,6 +97,16 @@ export default class MongoDBService {
     return this._currentConnectionOptions;
   }
 
+  get defaultConnectedDB(): string | null {
+    if (!this.connectionString) {
+      return null;
+    }
+    const connectionString = new ConnectionString(this.connectionString);
+    return connectionString.pathname !== '/'
+      ? connectionString.pathname.substring(1)
+      : null;
```

Comment:
We use this same code in `getActiveConnectionDefaultDB`, how do you feel about adding a shared helper somewhere?
I'm thinking then we can remove the `getActiveConnectionDefaultDB` from connection controller and use only the `getActiveConnectionString` to keep the connection controller interface a bit smaller.

Reply:
yea that sounds about right, i will take care of that.

Reply:
addressed in [cda24a](https://github.com/mongodb-js/vscode/pull/621/commits/cda24a93c4de52512432be2733da2a168bf20979)